File: kattis/fizzbuzz.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  int F, B, N;
  cin >> F >> B >> N;

  for (int i = 1; i <= N; i++) {
    if (i % F == 0 && i % B == 0)
      cout << "FizzBuzz";
    else if (i % F == 0)
      cout << "Fizz";
    else if (i % B == 0)
      cout << "Buzz";
    else
      cout << i;

    cout << endl;
  }
}

File: leetcode/fizz-buzz.go
package main

import "fmt"

func fizzBuzz(n int) []string {
	var res []string

	for i := 1; i <= n; i++ {
		var s string

		if i%3 == 0 {
			s += "Fizz"
		}

		if i%5 == 0 {
			s += "Buzz"
		}

		if len(s) == 0 {
			s = fmt.Sprint(i)
		}

		res = append(res, s)
	}

	return res
}
